### PR TITLE
add admission plugins CRD

### PR DESCRIPTION
**Pull request by zreigz**
**What this PR does / why we need it**: add admission plugins CRD. User can specify additional admission plugins in the Kubermatic which can be accessible from the API/UI

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

```release-note
dd admission plugins CRD support
```

### CODE REVIEW

File: api/pkg/crd/kubermatic/v1/admission_plugin.go [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+// AdmissionPlugin is the type representing a AdmissionPlugin
+type AdmissionPlugin struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	Spec AdmissionPluginSpec `json:"spec"`
+}
+
+// AdmissionPluginSpec specifies admission plugin name and from which k8s version is supported.
+// The FromVersion flag can be empty. It means the plugin fit to all k8s versions
```
Comment (xrstf) [inline api/pkg/crd/kubermatic/v1/admission_plugin.go]: Can you move this line down above the actual FromVersion field, so Go recognizes it as a docstring for that field? And rephrase it `FromVersion can be ....`. :-)

File: api/pkg/crd/kubermatic/v1/admission_plugin.go [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	Spec AdmissionPluginSpec `json:"spec"`
+}
+
+// AdmissionPluginSpec specifies admission plugin name and from which k8s version is supported.
+type AdmissionPluginSpec struct {
+	PluginName string `json:"pluginName"`
+
+	// The FromVersion flag can be empty. It means the plugin fit to all k8s versions
```
Comment (xrstf) [inline api/pkg/crd/kubermatic/v1/admission_plugin.go]: https://blog.golang.org/godoc-documenting-go-code

> Notice this comment is a complete sentence that **begins with the name of the element it describes**.

Set let's get rid of the "The " part.

Comment (zreigz) [reply api/pkg/crd/kubermatic/v1/admission_plugin.go]: done

**Comment by zreigz** [comment]
@xrstf thanks for your time and review :+1:

**Pull request merged**